io/fv3atm_restart_io.F90

```diff
@@ -644,8 +646,21 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain, warm_sta
     amiopen=open_file(Sfc_restart, trim(infile), "read", domain=fv_domain, is_restart=.true., dont_add_res_to_filename=.true.)
     if( .not.amiopen ) call mpp_error(FATAL, 'Error opening file'//trim(infile))
 
+    if (global_att_exists(Sfc_restart, "file_version")) then
+      call get_global_attribute(Sfc_restart, "file_version", file_ver)
+      if (file_ver == "V2") then
+        sfc%is_v2_file=.true.
+      endif
+    endif
+
+    print*,'^^^^ process sfc file ',sfc%is_v2_file
```

Comment:
Looks like I left some diagnostic prints. Should I remove all of them?

Reply:
That would be appreciated. Thanks.